lecture-variables.html

```diff
@@ -119,11 +133,11 @@
 someNumber /= x
 </code></pre>
 
-
+(Optional) <b>Challenge</b>: what is the value of <code>someNumber</code> at the end of the above code snippet? Walk through each line of code and try to compute the value of <code>someNumber</code> yourself. Then, paste the snippet above into codepen to check your answer.
```

Comment:
This should be wrapped in a `<p>` tag. I think i'll merge this and make the change on master in the interest of speed